cumulusci/tasks/metadeploy.py

```diff
@@ -112,10 +113,13 @@ def _run_task(self):
             version = self._find_or_create_version(product)
             if self.labels_path and "slug" in product:
                 self._publish_labels(product["slug"])
+        else:
+            version = {"id": "fake_for_dryrun", "url": "fake_for_dryrun"}
 
         # Check out the specified tag
         gh = self.project_config.get_github_api()
         repo = gh.repository(repo_owner, repo_name)
+        assert repo
```

Comment:
If repo is None, later code would crash, so this should be backwards compatible.